README.md

```diff
@@ -3,15 +3,17 @@
 [Deployed Link](https://tasterx.herokuapp.com/)
 
 ## Abstract
-TasteRX is an engaging UX born from the idea of curing the world of bad taste.  Do you have good taste in TV shows?  Do you think the world would be a better place if more people watched a particular show?  The TasteRX UI allows users to search for and then 'prescribe' a TV show with a custom message to a fellow human; the created prescription can then easily be shared via URL.   
+Do you have good taste in TV shows?  Do you think the world would be a better place if more people watched a particular show?  TasteRX allows users to search for and then 'prescribe' a TV show to another person. The created prescription can then easily be shared via URL.
```

Comment:
*tsk tsk* this was a perfect use case for a semicolon.  My wife even gave me a big kiss this morning when she saw I used it (she's an English teacher so she knows)